config/viper_test.go

```diff
@@ -0,0 +1,66 @@
+package config
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/spf13/viper"
+	"github.com/stretchr/testify/require"
+
+	_ "embed"
+)
+
+//go:embed test_files/config.json
+var configFile string
+
+func TestBuildConfig(t *testing.T) {
+	v := viper.New()
```

Comment:
We can get rid of `test_files/config.json` and re-use the top-level `sample-relayer-config.json` for this test. The following changes are necessary:
1. Read in `"../sample-relayer-config.json` using `os.ReadFile`. `go:embed` does not support `..`.
```
cfgBytes, err := os.ReadFile("../sample-relayer-config.json")
configFile := string(cfgBytes)
```
2. Modify the expected `DestinationBlockchain` `AccountPrivateKey` to match the partial key provided in `sample-relayer-config.json`
3. Remote `test_files/config.json`